Description: Add missing SOVERSION

--- idg-api-0.orig/CMakeLists.txt
+++ idg-api-0/CMakeLists.txt
@@ -106,7 +106,7 @@ add_library(
   ${PROJECT_NAME} ${${PROJECT_NAME}_headers}
                   ${${PROJECT_NAME}_sources}
 )
-set_target_properties(${PROJECT_NAME} PROPERTIES COMPILE_FLAGS "-fPIC")
+set_target_properties(${PROJECT_NAME} PROPERTIES COMPILE_FLAGS "-fPIC" SOVERSION 0)
 target_link_libraries (
     ${PROJECT_NAME} 
     ${${PROJECT_NAME}_link_libraries}
